data-structures-linked-lists/get-tail.js

```diff
@@ -0,0 +1,11 @@
+/* exported getTail */
+
+function getTail(list) {
+  let copy = list;
```

Comment:
Same here. When you assign a reference data type (anything that's not a "primitive") to a new variable, all you've done is create another name (aka alias) for the same thing in memory.

https://developer.mozilla.org/en-US/docs/Glossary/Primitive